airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -91,6 +91,9 @@ def get_connection_form_widgets() -> dict[str, Any]:
             "cluster_context": StringField(lazy_gettext("Cluster context"), widget=BS3TextFieldWidget()),
             "disable_verify_ssl": BooleanField(lazy_gettext("Disable SSL")),
             "disable_tcp_keepalive": BooleanField(lazy_gettext("Disable TCP keepalive")),
+            "xcom_sidecar_container_image": StringField(
```

Comment:
@dstandish 
Documentation was already included since the initial commit.
Is there anything else you suggest I add? 
Have a look [here](https://github.com/bdsoha/airflow/blob/feature/sidecar_container/docs/apache-airflow-providers-cncf-kubernetes/connections/kubernetes.rst#default-connection-ids).

Reply:
oh, forgive me, looked for it just didn't see it

Reply:
@dstandish Thanks for your guidance with this PR.

Reply:
of course, thanks for your contribution!